src/ripple/app/paths/cursor/DeliverNodeReverse.cpp

```diff
@@ -251,7 +252,14 @@ TER PathCursor::deliverNodeReverseImpl (
             resultCode = increment(-1).deliverNodeReverseImpl(
                 node().offerOwnerAccount_,
                 saInPassReq,
-                saInPassAct);
+                saInPassAct,
+                saOutAct > zero);
+
+            if (resultCode == tecPATH_DRY && saOutAct > zero)
+            {
+                resultCode = tesSUCCESS;
```

Comment:
@nbougalis: There are two results here: the result from the recursive call to `deliverNodeReverseImpl` on line 252 and the result from the current call to `deliverNodeReverseImpl` on line 260 (remember the call on line 252 is a 
recursive call.

Most of the time, the results are the same. This is one case where the recursive call can be tecPATH_DRY, but the current call is not tecPATH_DRY.

If we change the return value of the recursive call based on the context it's called in, we mix those two return values together and I think that makes it less clean.

@scottschurr: Added a comment